native/src/seal/encryptor.cpp

```diff
@@ -97,11 +101,12 @@ namespace seal
             parms_id, 
             is_asymmetric, 
             save_seed, 
-            false, 
-            false, 
+            false, // disable_special_modulus
```

Comment:
I'd expect you need to plumb something here.

Reply:
I have two different signatures for this function depending on if things are being exported. This is basically to keep the original semantics of the program and then I add my own parts on top.